helpers/helpers_test.go

```diff
@@ -1,13 +1,17 @@
 package helpers
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
 	"fmt"
 	"log"
 	"os"
 	"runtime"
 	"strings"
 	"testing"
+	"time"
 
+	"github.com/golang-jwt/jwt"
```

Comment:
```suggestion
	""github.com/lestrrat-go/jwx/v2/jwt""
```

Since you are doing a refactoring. Consider using the same JWT library used elsewhere in the code suite, it will make the merging easier down the road.